### PR TITLE
Added a simple web development template using Azure RM

### PR DESCRIPTION
**Pull request by predeekc**

### CODE REVIEW

File: app/models/jellyfish_azure/service/azure_service.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+module JellyfishAzure
+  module Service
+    class AzureService < ::Service::Storage
```
Comment (stackus) [inline app/models/jellyfish_azure/service/azure_service.rb]: May want to use just use `::Service` here. `Storage` may add undesired changes in the future.

File: app/models/jellyfish_azure/service/web_dev_environment.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          { label: 'US East', value: 'eastus' },
+          { label: 'US West', value: 'westus' }
+        ]
+      end
+
+      def provision
+        dns_name = settings[:az_dev_dns]
+
+        storage_account_check = check_storage_account dns_name
+        if not storage_account_check.name_available then
```
Comment (stackus) [inline app/models/jellyfish_azure/service/web_dev_environment.rb]: This line can be changed to `unless storage_account_check.name_available`

File: app/models/jellyfish_azure/service/web_dev_environment.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+          return
+        end
+
+
+        location = settings[:az_dev_location];
+        ensure_resource_group location
+
+        # template_url = File.expand_path('../../../../../templates/web-dev-environment/azuredeploy.json', __FILE__);
+        template_url = 'https://raw.githubusercontent.com/neudesic/jellyfish-azure/master/templates/web-dev-environment/azuredeploy.json';
```
Comment (stackus) [inline app/models/jellyfish_azure/service/web_dev_environment.rb]: We'll want this to point to the main repo.

File: public/extensions/azure/components/forms/fields.config.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+    Forms.fields('azure_storage_name', {
+      type: 'text',
+      templateOptions: {
+        label: 'Name'
+      },
+      validators: {
+        available: {
+          expression: function (viewModel, modelValue, scope) {
+            debugger;
```
Comment (stackus) [inline public/extensions/azure/components/forms/fields.config.js]: WIP?

Comment (predeekc) [inline public/extensions/azure/components/forms/fields.config.js]: Yeah, this needs to go away.

**Comment by predeekc** [comment]
I've performed the updates requested

**Pull request merged**